org.dash.dash-core.json

```diff
@@ -0,0 +1,87 @@
+{
+    "id": "org.dash.dash-core",
+    "runtime": "org.freedesktop.Platform",
+    "sdk": "org.freedesktop.Sdk",
+    "runtime-version": "24.08",
+    "command": "dash-entrypoint.sh",
+    "finish-args": [
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--device=dri",
+        "--share=ipc",
+        "--share=network",
+        "--filesystem=~/.dashcore:create",
+        "--filesystem=/run/tor/control.authcookie:ro"
+    ],
+    "modules": [
+        {
+            "name": "dash-core",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -m 0744 -D -t /app/bin/internal bin/dashd",
+                "install -m 0744 -D -t /app/bin/internal bin/dash-qt",
+                "install -m 0744 -D -t /app/bin/internal bin/dash-cli",
+                "install -m 0744 -D -t /app/bin/internal bin/dash-tx",
+                "install -m 0744 -D -t /app/bin/internal bin/dash-wallet",
+                "install -m 0744 -D -t /app/bin/internal bin/test_dash"
+            ],
+            "post-install": [
+                "/app/bin/internal/test_dash"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "only-arches": [
+                        "x86_64"
+                    ],
+                    "url": "https://github.com/dashpay/dash/releases/download/v21.1.1/dashcore-21.1.1-x86_64-linux-gnu.tar.gz",
+                    "sha256": "c3157d4a82a3cb7c904a68e827bd1e629854fefcc0dcaf1de4343a810a190bf5"
+                },
+                {
+                    "type": "archive",
+                    "only-arches": [
+                        "aarch64"
+                    ],
+                    "url": "https://github.com/dashpay/dash/releases/download/v21.1.1/dashcore-21.1.1-aarch64-linux-gnu.tar.gz",
+                    "sha256": "df54756e801d85d2e37ea76ff001692e897dd39b3479a5c7c4a6530da04ea4a5"
+                }
+            ]
+        },
+        {
+            "name": "misc",
+            "buildsystem": "simple",
+            "build-commands": [
+                "desktop-file-edit --set-key=\"Icon\" --set-value=${FLATPAK_ID} dash-qt.desktop",
+                "install -Dp -m 644 dash-qt.desktop /app/share/applications/${FLATPAK_ID}.desktop",
+                "install -Dp -m 644 dash.png /app/share/icons/hicolor/scalable/apps/org.dash.dash-core.png",
```

Comment:
scalable is not PNG

Reply:
https://github.com/flathub/flathub/pull/5747/commits/bdc3b760a5ef05028f1f75b5d59ea9bca3f98ebe